doc/sphinx/source/quickstart/installation.rst

```diff
@@ -640,8 +640,8 @@ again.
 
 .. _install_on_hpc:
 
-Pre-installed versions on HPC clusters
-======================================
+Pre-installed versions on HPC clusters / other servers
+======================================================
 
 The ESMValTool is also available on the HPC clusters CEDA-JASMIN and DKRZ-MISTRAL and there will be no need
 to install it yourself if you are just running diagnostics:
```

Comment:
Adding the Met Office to the opening sentence, and correcting the name of the HPC cluster at DKRZ:

```suggestion
ESMValTool is available on the HPC clusters CEDA-JASMIN and DKRZ-Levante, and on the Met Office Linux
estate, so there is no need to install ESMValTool if you are just running recipes:
```